applications-frontend/src/components/Upload.vue

```diff
@@ -0,0 +1,84 @@
+<template>
+  <div class="container">
+    <progress class="loading is-primary" v-if="waiting" max="100"></progress>
+    <form v-else class="form" enctype="multipart/form-data" method="POST">
+      <div class="field">
+        <label class="label">Applicants: </label>
+        <div class="file has-name">
+          <label class="file-label">
+            <input id="file" class="file-input" type="file" @change="fileChanged" required name="resume" />
```

Comment:
`resume` shouldn't be the name of the element. Maybe `applicants_spreadsheet`?

Reply:
name isn't used